extensions/2.0/Vendor/MSFT_RigidBodies/schema/node.MSFT_RigidBodies.rigidBody.schema.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "MSFT_RigidBodies glTF extension",
+    "type" : "object",
+    "description" : "Parameters describing how a node's transform should be driven by physics simulation.",
+    "allOf" : [ { "$ref" : "glTFChildOfRootProperty.schema.json" } ],
+    "properties" : {
+        "isKinematic" : {
+            "type": "boolean",
+            "description": "If enabled, the object effectively has infinite mass. It's velocity will remain constant during physics simulation.",
+            "default": false
+        },
+        "mass" : {
+            "type": "number",
+            "description": "The mass of the object.",
+            "default" : 1.0,
+            "minimum" : 0,
+            "exclusiveMinimum" : true
+        },
+        "centerOfMass" : {
+            "type": "array",
+            "description": "The center of mass of the object in local space.",
+            "items": {
+                "type": "number"
+            },
+            "minItems": 3,
+            "maxItems": 3,
+            "default": [ 0.0, 0.0, 0.0 ]
+        },
+        "inertiaTensor" : {
+            "type": "array",
+            "description": "A 3x3 matrix describing the mass distribution of the object in local space.",
+            "items": {
+                "type": "number"
+            },
+            "minItems": 9,
+            "maxItems": 9,
+            "default": [
+                1.0, 0.0, 0.0,
+                0.0, 1.0, 0.0,
+                0.0, 0.0, 1.0 ]
+        },
+        "linearVelocity" : {
+            "type": "array",
+            "description": "The initial linear velocity of the object.",
+            "items": {
+                "type": "number"
+            },
+            "minItems": 3,
+            "maxItems": 3,
+            "default": [ 0.0, 0.0, 0.0 ]
+        },
+        "angularVelocity" : {
+            "type": "array",
+            "description": "The initial angular velocity of the object.",
+            "items": {
+                "type": "number"
+            },
+            "minItems": 3,
+            "maxItems": 3,
+            "default": [ 0.0, 0.0, 0.0 ]
+        },
+        "gravityFactor" : {
+            "type": "number",
+            "description": "A multiplier applied to the scene's gravity.",
+            "default": 1.0
+        }
```

Comment:
Is it standard to have a `gravityFactor` defined for individual rigid bodies? I'm not super familiar with the physics side of things, but I would have thought gravity is set for the entire world and `mass` is how gravitational effects are altered for individual rigid bodies.

Reply:
Godot has it, but it's a really non-important feature, and I don't know if it's a good thing to standardize since some engines may not have it.

Reply:
That's a common mistake to make, but the mass of a body has absolutely zero effect on the gravitation acceleration it experiences.

While a non-unity value of this parameter is not physically realistic (exotic, theoretical particles not withstanding :)), I see games use it very frequently. It's simple to implement and allows you do a bunch of neat things (please excuse the programmer-art in the examples below):

Here, the particles which skin the cactus have a negative gravity factor:

https://user-images.githubusercontent.com/1676295/220128444-70e165fe-8c34-4e78-a131-a1af75abe501.mp4

It's also one of the tricks you'll need if you want to get "slow motion" objects even when the physics solver may not explicitly support it:

https://user-images.githubusercontent.com/1676295/220128894-e215ac94-66df-45ee-aa3c-f5a03c32b315.mp4

Reply:
> mass of a body has absolutely zero effect on the gravitation acceleration it experiences

Whoops, I understand my mistake. Need to brush up on my physics. 

As for the gravity factor (and other features), what about creating a feature support table for current physics/game engines so that we can evaluate a feature's prevalence?

Reply:
If this feature is common in physics engines, I would be happy to add it to OMI's rigid body spec.